numpy-stubs/__init__.pyi

```diff
@@ -450,6 +451,11 @@ def array(
     subok: bool = ...,
     ndmin: int = ...,
 ) -> ndarray: ...
+def asarray(
```

Comment:
The lack of a newline is really triggering my OCD here..

Reply:
I'm sorry to say that the official numpy stubs repo uses this formatting, plus [black](https://github.com/python/black) for enforcing consistent formatting.

We could do nicer formatting/change the rules for how we want to format things... but then it would make it more difficult if we wanted to try to push things upstream to the official repo.

Reply:
Oh no, this is filthy! But agreed, let's stick with the project conventions